File: app/src/main/java/vergecurrency/vergewallet/view/base/BaseFragment.java
package vergecurrency.vergewallet.view.base;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import vergecurrency.vergewallet.helpers.utils.UIUtils;
import vergecurrency.vergewallet.service.model.PreferencesManager;

public class BaseFragment extends Fragment {
	@Nullable
	@Override
	public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
		UIUtils.setTheme(PreferencesManager.getCurrentTheme(),getContext(),false);
		return super.onCreateView(inflater, container, savedInstanceState);
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/view/base/BaseFragment.kt
package vergecurrency.vergewallet.view.base

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

import vergecurrency.vergewallet.helpers.utils.UIUtils
import vergecurrency.vergewallet.service.model.PreferencesManager

open class BaseFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        UIUtils.setTheme(PreferencesManager.currentTheme!!, context!!, false)
        return super.onCreateView(inflater, container, savedInstanceState)
    }
}
